vttools/vtmods/import_lists/modules.yaml

```diff
@@ -49,9 +49,20 @@ autowrap_func:
 - func_name : pixel_to_phi
   module_path: skxray.core
   namespace: core
+# IO
```

Comment:
Yes sorry. I was changing this file(yaml) try add those then I figured out it is not working. Then I saw that some things already not even merged(skxray/core) in my current branch is in vistrails. Then after chating  with @tacaswell  and @giltis  I got to know that I have to change vt_config/NSLS-II/init.py. I will delete this part